### PR TITLE
add AWS v16.4.2

### PR DESCRIPTION
**Pull request by kubasobon**

### CODE REVIEW

File: aws/v16.4.2/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# :zap: Giant Swarm Release v16.4.2 for AWS :zap:
+
+<< Add description here >>
```
Comment (alex-dabija) [inline aws/v16.4.2/README.md:3]: @kubasobon, @gianfranco-l we need a description for the release in order to make it available to our customers. Could you provide one, given that your fix is the only change in this release? Thanks!

Comment (gianfranco-l) [reply aws/v16.4.2/README.md:3]: @alex-dabija created related [pr here](https://github.com/giantswarm/releases/pull/989/files) cc @ljakimczuk  @piontec @MarcelMue

**Review by gianfranco-l — APPROVED** [review]

File: aws/v16.4.2/README.md (hunk 1 above)
Comment (alex-dabija) [reply aws/v16.4.2/README.md]: Thanks! I tweaked it a bit and added the release announcement.

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**